app/views/pages/home/index.blade.php

```diff
@@ -1,30 +1,30 @@
 @section('content')
     <div class="ui blue inverted segment">
-        <h3 style="text-align: center;">About Campaign Crawler</h3>
+        <h3 style="text-align: center; margin-top:0px;">About Campaign Crawler</h3>
```

Comment:
Inline styles are normally a nono, but i'll let it slide since it's so close to presentation